components/homme/src/tool/tool_main.F90

```diff
@@ -0,0 +1,109 @@
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+! This program is an offline tool. See the documentation in the following
+! subroutines, implemented below; each is a tool:
+!   * topo_convert
```

Comment:
Maybe I'm just not understanding this, but the name "tool_main" seems very generic. Can we be more descriptive of what this is actually used for? It seems to be an offline file processing tool, would that be accurate?

Reply:
Yes. Homme previously did not have a clear offline tool system, just workarounds. This new executable will be the place for any new offline tool to live. How about the name offline_tool_main? (I'd like to keep the directory name as 'tool' because Homme directory names tend to be short, which is kind of nice.)

Reply:
oh ok, since it really is a general offline tool then a general name makes sense. I was just worried it was more specific than that. so "tool_main" is fine.